webapp/channels/src/components/channel_info_rhs/header.tsx

```diff
@@ -20,8 +20,11 @@ const Icon = styled.i`
     font-size:12px;
 `;
 
-const HeaderTitle = styled.span`
+const HeaderTitle = styled.h2`
```

Comment:
nit: the plan is start removing styled components from the code base , if possible when dealing with one we should just end its misery.